### DISCUSSION TITLE
Can Quarto convert word and odf documents the same way fastpages could?

### DISCUSSION THREAD

**Discussion by moonpiedumplings** [quarto-dev/quarto-cli#5915] [Q&A]
### Description

See here: https://github.com/fastai/fastpages#writing-blog-posts-with-microsoft-word

I am wondering if quarto has a similar feature.

**Comment by mcanouil** [comment]
Quarto is based on Pandoc, and Pandoc can, see <https://pandoc.org/MANUAL.html#option--from>.
You need to tell Pandoc what is what from the command line.

The thing is, and I am pretty sure, you'll lose a bunch of features from Quarto.
Also, Pandoc is pretty strict on what it can understand from Word document. Basically, if you made a Word that is not based on Pandoc Word template, it might not work very well.

May I ask what's your goal here?

**Reply by moonpiedumplings** [reply]
I'd like to write odf documents in libreoffice, taking advantage of the features of a full word processor like advanced spelling and grammar checking, but then get it converted to a webpage in my blog.

**Reply by mcanouil** [reply]
Depending on your IDE, you can have that.

Most likely your Word document will be made with Word thus could differ from Pandoc default Word document leading to a broken workflow.
I fear Quarto/Pandoc might not meet your workflow needs.
